### ISSUE TITLE
Link not accessible via Tabs in contact form

### ISSUE THREAD

**Issue by hmkay** [liqd/digitalstrategie#256]
**URL:** https://digitalstrategie-dev.liqd.net/kontakt/
**user:** participant
**expected behaviour:** I can tick the box to have read the data protection and access the data protection 
**behaviour:** I cannot access the link of the data protection in the contac form
**important screensize:** desktop
**device & browser:** MacOS Monterey
**Comment/Question:** Actually the same counts for the participation form

![Bildschirmfoto 2022-02-14 um 17 57 11](https://user-images.githubusercontent.com/30862422/153910751-7e4c5e2b-0497-4cb8-aaef-5e0379f82ebf.png)

**Comment by phillimorland** [comment]
@hmkay so different page elements require different keyboard shortcuts, to tick a checkbox or radio you must press space bar, to read an element that is not a heading (the link) again it's spacebar or i think ctrl and right, i've checked this in ff and chrome and they both work, are you using a browser or mac screen reader or anything?

**Comment by hmkay** [comment]
ah, that makes sense, thanks for the clarification, @phillimorland ! I also used Firefox and still didn´t find the right combination to tab to the Datenschutz, but I am sure that is on me. On my way to learn testing a11y...

**Comment by phillimorland** [comment]
@hmkay it might also be a mac issue although they are in general much better then linux for a11y, I'll double check on the owl mac later!

**Comment by hmkay** [comment]
Thank you!!

**Comment by phillimorland** [comment]
@hmkay ok you should have a look at this, I forgot about it but Caro and me had this issue a while back https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/ will add to the wiki, it now works on powl mac

**Comment by hmkay** [comment]
Awesome, now it also works for me, thanks for the hint, @phillimorland !
